docs/core/setup/includes/package-manager-heading-hack-pkgname.md

```diff
@@ -0,0 +1,31 @@
+
+The packages added to feeds are named in a hackable format: `{product}-{type}-{version}`.
+
+- **product**\
+The type of .NET product to install. Valid options are:
+
+  - dotnet
+  - aspnetcore
+
+- **type**\
+Chooses the SDK or the runtime. Valid options are:
+
+  - sdk
+  - runtime
+
+- **version**\
```

Comment:
Here we should also include info on installing a specific patch. 
* On Ubuntu and Debian the format is `{product}-{type}-{version}={patch}-1` (e.g. `dotnet-sdk-3.0=3.0.100-1`)
* There is no patch command for RHEL
* On other distros the format is `{product}-{type}-{version}-{patch}-1` (e.g. `dotnet-sdk-3.0-3.0.100-1`)

Reply:
Got this logged in the follow up issue.